core/src/main/scala/org/apache/spark/ui/exec/ExecutorsPage.scala

```diff
@@ -128,11 +152,36 @@ private[ui] class ExecutorsPage(
       <td sorttable_customkey={diskUsed.toString}>
         {Utils.bytesToString(diskUsed)}
       </td>
-      <td>{info.activeTasks}</td>
-      <td>{info.failedTasks}</td>
-      <td>{info.completedTasks}</td>
+      <td style={
+        if (info.activeTasks > 0) {
+          "background:hsla(120, 100%, 25%, " + activeTasksAlpha + ");color:white"
```

Comment:
Can the `style="..."` text go inside the conditional so we don't end up with `style=""` or does that not work in this syntax?